apps/ftu/js/language.js

```diff
@@ -8,13 +8,34 @@ var LanguageManager = {
   init: function init() {
     this.buildLanguageList();
     document.getElementById('languages').addEventListener('change', this);
-    this.settings.addObserver('language.current',
-      function updateDefaultLayouts(event) {
-        // the 2nd parameter is to reset the current enabled layouts
-        KeyboardHelper.changeDefaultLayouts(event.settingValue, true);
-      });
     window.addEventListener('localized',
```

Comment:
Nit: if we're going to uninit, lets remove the localized and change event listeners in uninit too